eng/pipelines/templates/stages/platform-matrix.json

```diff
@@ -8,15 +8,15 @@
       "windows-2019": { "OSVmImage": "MMS2019", "Pool": "azsdk-pool-mms-win-2019-general" },
       "macOS-10.15": { "OSVmImage": "macOS-10.15", "Pool": "Azure Pipelines" }
     },
-    "PythonVersion": [ "pypy3", "2.7", "3.6", "3.7", "3.8" ],
+    "PythonVersion": [ "pypy3", "3.6", "3.7", "3.8", "2.7" ],
```

Comment:
Lets not rely on our sparse ordering to workaround an issue just for identity and keyvault. Lets be explicit and deterministic and do the exclude/include in those 2 pipelines like you did for macos.